### PR TITLE
YSHOP2-861: Products list

### PR DESCRIPTION
**Pull request by adildev101**
## JIRA Ticket

[YSHOP2-861](https://youcanshop.atlassian.net/browse/YSHOP2-861)

## QA Steps

-  [ ] `pnpm i`
-  [ ] `pnpm dev`

## Note

Leave empty when you have nothing to say

[YSHOP2-861]: https://youcanshop.atlassian.net/browse/YSHOP2-861?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: sections/main-collection.liquid
Comment (eihabkhan1) [inline sections/main-collection.liquid:1]: We should use the reusable snippet from cod-theme-3

File: sections/main-collection.liquid [hunk 1]
```diff
@@ -3,37 +3,25 @@
...
-      </div>
-      <div class="listings__item">
-        <a href="/collections" class="second-listing">{{ 'snippets.collection-listing.collections' | t }}</a>
-      </div>
-    </div>
-
-    <div class="breadcrumbs-name">
-      <h1>{{ category.name }}</h1>
+  <div class="collection-heading">
+    {%- render 'breadcrumbs' -%}
```
Comment (eihabkhan1) [inline sections/main-collection.liquid:7]: We should use the reusable snippet from cod-theme-3

**Review by eihabkhan1 — APPROVED** [review]
Thank you for making the changes @adildev101 🙌

File: sections/main-collection.liquid (hunk 1 above)
Comment (adildev101) [reply sections/main-collection.liquid:7]: in this iteration we want to have the same code as the theme 1 so about the breadcrumbs we already have a ticket for it so we will handle it on the cod theme 1 and theme 2

**Pull request merged**